Classes/Models/Singletons/SavedSettings.swift

```diff
@@ -125,6 +125,9 @@ final class SavedSettings {
         }
     }
     
+    @UserDefault(key: .isPlaying, defaultValue: false)
+    var isPlaying: Bool
```

Comment:
It looks like this needs to be set to true/false by the player or this will always be false. Also it would need to be reset to false on app launch or it will be true if the user was playing music when the closed the app even if the player isn't playing anymore.

Reply:
Changes requested at the meeting  implemented.